README.md

```diff
@@ -69,8 +84,9 @@ Using this method, you can open the monitor like so:
 ```js
```

Comment:
Nit -- the script above actually opens the Video Room Monitor, so maybe this could say something like:

This script will open the monitor. Using this method, you can then close or toggle the monitor like so:
```
Twilio.VideoRoomMonitor.closeMonitor();
Twilio.VideoRoomMonitor.toggleMonitor();
```

Also, could the note above specify that the Room object needs to be exposed with the name `twilioRoom`, or they should substitute that variable name with whatever they've called the global room variable?